l10n_it_fatturapa_in/views/account_view.xml

```diff
@@ -139,6 +140,9 @@
                     <field name="virtual_stamp" readonly="1"></field>
                     <field name="stamp_amount" readonly="1"></field>
                 </field>
+                <field name="invoice_line" position="attributes">
+                    <attribute name="context">{'form_view_ref' : 'l10n_it_fatturapa_in.view_invoice_line_form_fatturapa_in'}</attribute>
```

Comment:
2015-06-10 14:12 GMT+02:00 Alessio Gerace <notifications@github.com>:

> In l10n_it_fatturapa_in/views/account_view.xml
> <https://github.com/eLBati/l10n-italy/pull/63#discussion_r32110618>:
>
> > @@ -139,6 +140,9 @@
> >                      <field name="virtual_stamp" readonly="1"></field>
> >                      <field name="stamp_amount" readonly="1"></field>
> >                  </field>
> > +                <field name="invoice_line" position="attributes">
> > +                    <attribute name="context">{'form_view_ref' : 'l10n_it_fatturapa_in.view_invoice_line_form_fatturapa_in'}</attribute>
>
> ho diviso i due form, perche' in fattura PA in deve essere tutto readonly
> , mentre in fattura PA out devono essere scrivibili quei campi..
>
>
>
Ma in questo modo mi pare che se installi entrambi i moduli, il campo
admin_ref è sempre readonly, no?
Io lo aggiungerei una volta sola e lo lascerei sempre modificabile.
Consideralo come gli altri campi come la descrizione della riga fattura